### PR TITLE
Add voluntary exit processing to E2E

### PR DESCRIPTION
**Pull request by 0xKiwi**
**What type of PR is this?**
E2E test addition

**What does this PR do? Why is it needed?**
This PR adds voluntary exits to E2E simulating a validator submitting an exit, and ensuring the validator is marked as exited in the state.

### CODE REVIEW

File: endtoend/evaluators/operations.go [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+	activeEpoch11Count := 0
+	activeEpoch12Count := 0
+	activeEpoch13Count := 0
+	exitEpochWrongCount := 0
+	withdrawEpochWrongCount := 0
+	for _, item := range validators.ValidatorList {
+		if item.Validator.EffectiveBalance < params.BeaconConfig().MaxEffectiveBalance {
+			effBalanceLowCount++
+		}
+		if item.Validator.ActivationEpoch == 10 {
```
Comment (terencechain) [inline endtoend/evaluators/operations.go:89]: Would a switch statement be better here?

File: endtoend/evaluators/operations.go [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+		}
+	}
+
+	if effBalanceLowCount > 0 {
+		return fmt.Errorf(
+			"%d validators did not have genesis validator effective balance of %d",
+			effBalanceLowCount,
+			params.BeaconConfig().MaxEffectiveBalance,
+		)
+	} else if activeEpoch10Count != int(churnLimit) {
```
Comment (terencechain) [inline endtoend/evaluators/operations.go:113]: same here, switch statement?

File: endtoend/evaluators/operations.go [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+	"github.com/prysmaticlabs/prysm/beacon-chain/core/helpers"
+	e2e "github.com/prysmaticlabs/prysm/endtoend/params"
+	"github.com/prysmaticlabs/prysm/endtoend/types"
+	"github.com/prysmaticlabs/prysm/shared/params"
+	"github.com/prysmaticlabs/prysm/shared/testutil"
+	"golang.org/x/exp/rand"
+	"google.golang.org/grpc"
+)
+
+var exitedIndice = params.BeaconConfig().FarFutureEpoch
```
Comment (rauljordan) [inline endtoend/evaluators/operations.go:19]: Is there a better way to represent this? also naming could be different + have a comment

File: endtoend/evaluators/operations.go [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+	signingData, err := helpers.ComputeSigningRoot(voluntaryExit, domain.SignatureDomain)
+	if err != nil {
+		return err
+	}
+	signature := privKeys[exitedIndice].Sign(signingData[:])
+	signedExit := &eth.SignedVoluntaryExit{
+		Exit:      voluntaryExit,
+		Signature: signature.Marshal(),
+	}
+	_, err = valClient.ProposeExit(ctx, signedExit)
```
Comment (rauljordan) [inline endtoend/evaluators/operations.go:208]: if _, err := ...; err != nil

File: endtoend/evaluators/operations.go (hunk 2 above)
Comment (0xKiwi) [reply endtoend/evaluators/operations.go]: Can't really make it work for this one but worked great for the first one, thanks :+1:

**Pull request merged**